app/views/clients/rollup/_assessments.html.haml

```diff
@@ -88,7 +88,7 @@
                   %span.icon-plus
           - assessments.each do |assessment|
             %tr{ class: cz, style: 'display:none;' }
-              %td= link_to_if can_view_enrollment_details_tab? name, client_hud_assessment_path(@client, assessment), class: 'btn btn-primary btn-muted btn-xs', data: { loads_in_pjax_modal: true }
+              %td= link_to name, client_hud_assessment_path(@client, assessment), class: 'btn btn-primary btn-muted btn-xs', data: { loads_in_pjax_modal: true }
```

Comment:
I think we no longer need this now that can view clients grants access to it? not sure, please check me @eanders . This was throwing an error.

Reply:
Oh, looks like I was missing a comma.  I think you want to leave in the check, but I'll push a fix.